src/Bicep.Core.Samples/Files/Functions/az.json

```diff
@@ -10,7 +11,8 @@
   },
   {
     "name": "environment",
-    "fixedParameterTypes": [],
+    "description": "Returns information about the Azure environment used for deployment.",
+    "fixedParameters": [],
     "minimumArgumentCount": 0,
```

Comment:
>minimumArgumentCount [](start = 5, length = 20)

Some notes for reviewing this files:
- function names, min, max, types, flags should stay the same
- type signatures should have parameter name changes